docs/db-migration/prototypes/admin-settings.html

```
@@ -0,0 +1,523 @@
+        </div>
+    </header>
+
+    <main class="max-w-7xl mx-auto px-6 py-6">
+        <div class="mb-6">
+            <h2 class="text-2xl font-semibold tracking-tight">Admin Settings</h2>
+            <p class="text-sm text-muted mt-1">Configure field visibility, default values, and deployment-specific behavior.</p>
+        </div>
+
+        <!-- Tabs -->
+        <div class="flex gap-6 mb-6" style="border-bottom:1px solid var(--border)" id="tabs">
+            <button class="tab-active pb-3 text-sm font-medium" onclick="showTab('fields')">Field Configuration</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)